File: cases/taint/c/call-context-negative/call_context_negative.c
int dfb_source(void) { // DFB-SOURCE: call-context-input
    return 1;
}

int relay(int value) { // DFB-WITNESS: call-context-relay
    return value;
}

void dfb_sink(int value) {} // DFB-SINK: call-context-sink

void run(void) {
    int tainted = relay(dfb_source());
    int clean = relay(0);
    dfb_sink(clean);
}

File: cases/taint/rust/call-context-negative/call_context_negative.rs
fn dfb_source() -> i32 { // DFB-SOURCE: call-context-input
    1
}

fn relay(value: i32) -> i32 { // DFB-WITNESS: call-context-relay
    value
}

fn dfb_sink(value: i32) {} // DFB-SINK: call-context-sink

fn run() {
    let tainted = relay(dfb_source());
    let clean = relay(0);
    dfb_sink(clean);
}
